public_html/application/classes/Model/Batch.php

```diff
@@ -680,7 +680,7 @@ public static function mergekingdoms($securitykey=null, $kingdomsourcename, $kin
 
 				$sql = "update kingdoms set status = 'deleted' where name = {$kingdomtargetname}";
```

Comment:
I should have been more clear in the description, the `Database::QUERYTYPE` depends on the sql query. This one for instance needs to be `Database::UPDATE`